codespell_lib/data/dictionary_rare.txt

```diff
@@ -2,6 +2,7 @@ afterwords->afterwards
 amination->animation, lamination,
 aminations->animations, laminations,
 aminator->animator, laminator,
+aminators->animators, laminators,
```

Comment:
@luzpaz are these both valid, I could only find the others.

Reply:
Are you asking whether _animators_ and _laminators_ are valid? They are just the plural inflections of _[animator](https://www.collinsdictionary.com/dictionary/english/animator)_ and _[laminator](https://www.collinsdictionary.com/dictionary/english/laminate)_, I don't see how they cannot be valid.

Reply:
That said, it is true that _laminator_ is missing from the [Cambridge](https://www.google.com/search?q=site%3Adictionary.cambridge.org+laminator), [Oxford](https://www.google.com/search?q=site%3Aoxfordlearnersdictionaries.com+laminator), [MacMillan](https://www.google.com/search?q=site%3Amacmillandictionary.com+laminator) or [Longman](https://www.google.com/search?q=site%3Aldoceonline.com+laminator) online dictionaries, 
but not from the [Collins](https://www.collinsdictionary.com/dictionary/english/laminate), [Merriam-Webster](https://www.merriam-webster.com/dictionary/laminate) or [American Heritage](https://www.ahdictionary.com/word/search.html?q=laminator) dictionaries. The plural form is implicit, if you require explicit mentions of it, you will find occurrences in [Wikipedia](https://en.wikipedia.org/wiki/Lamination), [Vocabulary.com](https://www.vocabulary.com/dictionary/laminator), [The Free Dictionary](https://www.thefreedictionary.com/laminators) or [YourDictionary](https://www.yourdictionary.com/laminators).